ej2-angular/dialog/how-to/close-dialog-while-click-on-outside-of-dialog.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
-title: Close dialog while click on outside of dialog in Angular Dialog component | Syncfusion
-description: Learn here all about Close dialog while click on outside of dialog in Syncfusion Angular Dialog component of Syncfusion Essential JS 2 and more.
+title: Close dialog by clicking outside in Angular Dialog | Syncfusion
+description: Learn here all about close dialog by clicking outside in Syncfusion Essential Angular Dialog component, it's elements and more.
 platform: ej2-angular
 control: Close dialog while click on outside of dialog 
 documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Close dialog while click on outside of dialog in Angular Dialog component
+# Close by clicking outside in Angular Dialog Component
```

Comment:
Change like as: 
Close dialog while click on outside of Angular Dialog Component